base/docs/helpdb/Base.jl

```diff
@@ -2265,11 +2265,11 @@ sortperm
 """
     mod2pi(x)
 
-Modulus after division by 2pi, returning in the range \[0,2pi).
+Modulus after division by ``2π``, returning in the range ``[0,2π)``.
```

Comment:
latex math for the ranges since that notation isn't julia code, code for the rest